Exception in ASGI application - 'NoneType' object is not subscriptable

Hello, I have 'NoneType' error on asgi_send method, im not sure for why this happens:

```
[2018-11-07 22:13:39 -0500] [2868] [ERROR] Exception in ASGI application
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/uvicorn/protocols/websockets/websockets_impl.py", line 140, in run_asgi
    result = await asgi(self.asgi_receive, self.asgi_send)
  File "/usr/local/lib/python3.6/dist-packages/channels/sessions.py", line 175, in __call__
    return await self.inner(receive, self.send)
  File "/usr/local/lib/python3.6/dist-packages/channels/middleware.py", line 41, in coroutine_call
    await inner_instance(receive, send)
  File "/usr/local/lib/python3.6/dist-packages/channels/consumer.py", line 54, in __call__
    await await_many_dispatch([receive, self.channel_receive], self.dispatch)
  File "/usr/local/lib/python3.6/dist-packages/channels/utils.py", line 50, in await_many_dispatch
    await dispatch(result)
  File "/usr/local/lib/python3.6/dist-packages/channels/consumer.py", line 67, in dispatch
    await handler(message)
  File "/usr/local/lib/python3.6/dist-packages/channels/generic/websocket.py", line 173, in websocket_connect
    await self.connect()
  File "/home/deploy/xxxxxx/app/consumers.py", line 314, in connect
    await self.accept()
  File "/usr/local/lib/python3.6/dist-packages/channels/generic/websocket.py", line 186, in accept
    await super().send({"type": "websocket.accept", "subprotocol": subprotocol})
  File "/usr/local/lib/python3.6/dist-packages/channels/consumer.py", line 75, in send
    await self.base_send(message)
  File "/usr/local/lib/python3.6/dist-packages/channels/sessions.py", line 226, in send
    return await self.real_send(message)
  File "/usr/local/lib/python3.6/dist-packages/uvicorn/protocols/websockets/websockets_impl.py", line 170, in asgi_send
    self.scope["server"][0],
TypeError: 'NoneType' object is not subscriptable
```
Any ideas?

Django==2.1.3
channels==2.1.5
channels_redis==2.3.1
gunicorn==19.9.0
uvicorn==0.3.20

--- a/uvicorn/__init__.py
+++ b/uvicorn/__init__.py
@@ -1,4 +1,4 @@
 from uvicorn.main import main, run
 
-__version__ = "0.3.21"
+__version__ = "0.3.22"
 __all__ = ["main", "run"]

--- a/uvicorn/protocols/websockets/websockets_impl.py
+++ b/uvicorn/protocols/websockets/websockets_impl.py
@@ -167,7 +167,7 @@ class WebSocketProtocol(websockets.WebSocketServerProtocol):
             if message_type == "websocket.accept":
                 self.logger.info(
                     '%s - "WebSocket %s" [accepted]',
-                    self.scope["server"][0],
+                    self.scope["client"],
                     self.scope["path"],
                 )
                 self.initial_response = None
@@ -177,7 +177,7 @@ class WebSocketProtocol(websockets.WebSocketServerProtocol):
             elif message_type == "websocket.close":
                 self.logger.info(
                     '%s - "WebSocket %s" 403',
-                    self.scope["server"][0],
+                    self.scope["client"],
                     self.scope["path"],
                 )
                 self.initial_response = (http.HTTPStatus.FORBIDDEN, [], b'')

--- a/uvicorn/protocols/websockets/wsproto_impl.py
+++ b/uvicorn/protocols/websockets/wsproto_impl.py
@@ -215,7 +215,7 @@ class WSProtocol(asyncio.Protocol):
             if message_type == "websocket.accept":
                 self.logger.info(
                     '%s - "WebSocket %s" [accepted]',
-                    self.scope["server"][0],
+                    self.scope["client"],
                     self.scope["path"],
                 )
                 self.handshake_complete = True
@@ -231,7 +231,7 @@ class WSProtocol(asyncio.Protocol):
                 })
                 self.logger.info(
                     '%s - "WebSocket %s" 403',
-                    self.scope["server"][0],
+                    self.scope["client"],
                     self.scope["path"],
                 )
                 self.handshake_complete = True

